models/public/cocosnet/cocosnet.md

```diff
@@ -0,0 +1,130 @@
+# cocosnet
+
+## Use Case and High-Level Description
+
+Cross-domain correspondence network is a exemplar-based image translation model. \
+Model was pre-trained on ADE20k dataset. The `CoCosNet` architecture comprises two sub-networks: \
+`Correspondence` network and `Translation` network.
+For details see [paper](https://arxiv.org/pdf/2004.05571) and [repository](https://github.com/microsoft/CoCosNet).
+
+## Example
+
+## Composite model specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Source framework                | PyTorch*                                  |
+
+## Specification for Correspondence model
+
+The purpose of correspondence model is to establish correspondence between input image and given exemplar. \
+Correspondence network return warped exemplar with semantic from input.
+
+### Performance
+
+### Inputs
+
+1. name: "input.1", shape: [1x151x256x256] - Input semantic segmentation mask (one-hot label map) in the format [BxCxHxW],
+   where:
+    - B - batch size
+    - C - number of classes (151 for ADE20k)
+    - H - mask height
+    - W - mask width
+
+2. name: "input.70", shape: [1x3x256x256] - An reference image (exemplar) in the format [BxCxHxW],
```

Comment:
looks strange, possible it makes sense to add descriptive names (e.g. semantic and reference or something which hint which data model expects)  to model during conversion to onnx

Reply:
I add descriptive names, but then files on https://download.01.org should be replaced with https://drive.google.com/drive/folders/1FXgRWhHcHaN4TW--2msoejWCFGtiS_nr?usp=sharing